README.md

```diff
@@ -1,3 +1,5 @@
+[![Code Climate](https://codeclimate.com/github/alexandru-calinoiu/fog-akamai/badges/gpa.svg)](https://codeclimate.com/github/alexandru-calinoiu/fog-akamai)
```

Comment:
This url is wrong. Is pointing to your repository. It should be the one at the org `fog`. Can you fix it plz?

Thanks.